bin/skopos

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-source $SKOPOS
-skopos $@
+source "$SKOPOS"
```

Comment:
what if these are not defined?

Reply:
I'm not sure I understand what you mean by "these" in this context? This is a good change, though.